ttsfm/async_client.py

```
@@ -0,0 +1,438 @@
+"""
+Asynchronous TTS client implementation.
+
+This module provides the AsyncTTSClient class for asynchronous
+text-to-speech generation with OpenAI-compatible API.
+"""
+
+import json
+import uuid
+import asyncio
+import logging
+from typing import Optional, Dict, Any, Union, List
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)